lua/astrocommunity/pack/rust/rust.lua

```diff
@@ -18,7 +18,22 @@ return {
     "simrat39/rust-tools.nvim",
     ft = { "rust" },
     init = function() utils.list_insert_unique(astronvim.lsp.skip_setup, "rust_analyzer") end,
-    opts = function() return { server = require("astronvim.utils.lsp").config "rust_analyzer" } end,
+    opts = function()
+      local package_path = require("mason-registry.index.codelldb"):get_install_path()
+      local codelldb_path = package_path .. "/codelldb"
+      local liblldb_path = package_path .. "/extension/lldb/lib/liblldb"
+
+      if vim.loop.os_uname().sysname == "Linux" then
+        liblldb_path = liblldb_path .. ".so"
+      else
+        liblldb_path = liblldb_path .. ".dylib"
+      end
```

Comment:
Personally I prefer ternary operations for such a simple if-statement but that might be controversial so feel free to leave it as it is.

```suggestion
      liblldb_path = liblldb_path .. (vim.loop.os_uname().sysname == "Linux" and ".so" or ".dylib"))
```